### ISSUE TITLE
Error Resizing Images

### ISSUE THREAD

**Issue by MWalid** [fraywing/textAngular#648]
I am new to `textAngular` trying to get familiar with it, unforetenatly I get some errors with my local installation copy,

When I try to resize any image I cannot, I've noticed the following error in the console:
```
Uncaught ReferenceError: el is not defined
```

Which leads to this [line](https://github.com/fraywing/textAngular/blob/master/src/textAngular.js#L2048):
```javascript
el = angular.element(_el);
```

The reason of that error still unclear for me.

Lucky me, the solution was easy:
```javascript
var el = angular.element(_el);
```

The online [demo](http://textangular.com/) is working as expected,
AngularJs version: 1.3.15
textAngular version: 1.3.11

**Comment by figalex** [comment]
I just noticed the same issue on my local copy. The ```el``` variable is never declared. I don't know why the ```_el``` variable was not used.

**Comment by Clan-Utility** [comment]
Same issue happened here

**Comment by JoelParke** [comment]
Please retest with the forth coming release -- we hope to have that out in a few days.  I believe this is fixed.
If this is still and issue, please re-open.

**Comment by aelamel** [comment]
Any chance to fix this issue for Angular 1.2.x ?
Adding 'var' to the v1.3.11 version ?
